spec/lib/granite/projector/controller_actions_spec.rb

```diff
@@ -2,20 +2,26 @@
   prepend_before do
     stub_class(:projector, Granite::Projector) do
       get :confirm do
+        render json: {success: true, action: 'confirm'}
       end
 
       post :perform do
+        render json: {success: true, action: 'perform'}
       end
 
       get :custom, as: '' do
+        render json: {success: true, action: 'custom'}
       end
 
       post :custom_post, as: '' do
+        render json: {success: true, action: 'custom_post'}
       end
     end
     stub_class(:descendant, Projector)
-    stub_class(:action, Granite::Action) do
+    stub_class(:dummy_action, Granite::Action) do
```

Comment:
Renamed to `DummyAction`, because controller specs are passing `action: '...'` in params when initializing the BA. However since the key matches `action_name`, it tries to merge the initialization attributes with `attributes[:action]` in https://github.com/toptal/granite/blob/383ed072e1d0bf3ae1d3262eb5a4f75c8767876f/lib/granite/action.rb#L37